at_commons/lib/src/keystore/at_key.dart

```diff
@@ -151,6 +167,23 @@ class AtKey {
       ..namespace(namespace);
   }
 
+  /// Local key are confined to the client(device)/server it is created.
+  /// The key does not sync between the local-secondary and the cloud-secondary.
+  ///
+  /// Builds a local key and return a [LocalKeyBuilder].
+  ///
+  /// Example: local:phone.wavi@alice
+  /// ```dart
+  /// AtKey localKey = AtKey.local('phone',namespace:'wavi').build();
+  /// ```
+  static LocalKeyBuilder local(String key,
+      {String? namespace, String sharedBy = ''}) {
```

Comment:
Should sharedBy be mandatory parameter? Local keys without sharedBy will not match the regex.

Reply:
Made sharedBy as mandatory